app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -544,12 +519,12 @@ const CallScreen = ({componentId, currentCall, participantsDict, teammateNameDis
                         <Pressable
                             testID={'toggle-speakerphone'}
                             style={style.button}
-                            onPress={() => setSpeakerphoneOn(!currentCall?.speakerphoneOn)}
+                            onPress={() => setSpeakerphoneOn(!currentCall.speakerphoneOn)}
```

Comment:
not a fan of in-line functions, but 0/5 if you want to change it.

Reply:
Good point, fixed!